.github/workflows/nightly-builds.yml

```diff
@@ -15,11 +15,15 @@ on:
     paths:
       - .github/workflows/nightly-builds.yml
   pull_request:
-    paths:
-      - .github/workflows/nightly-builds.yml
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
```

Comment:
So we'll now need to explicitly label PRs that touch the nightly workflow if we want these to run on the PR?

Reply:
That's correct. I feel like there are more cases where we'd like to manually run the workflow than times when we modify the workflow itself, which we can still trigger with this.

Two other changes:
1. The reporting part of the workflow only runs on failure--it used to run on all PRs that modified the file. Checking that is harder now, so I opted to just run on failure. Otherwise, it would run on *every* PR (though it wouldn't actually open an issue)
2. Because we're now triggering on more PRs and then filtering with conditions, the nightly checks will show up (though skipped by default) on every PR. I'm going to look to see if there's anything to be done, but I doubt it.